README.md

```diff
@@ -60,7 +60,7 @@ The `configlet sync` command replaces the functionality of the older `canonical_
 
 ## Use in your track
 
-To use the application in your track, you can copy the [`scripts/fetch-configlet`](./scripts/fetch-configlet) and/or [`scripts/fetch-configlet.ps1`](./scripts/fetch-configlet.ps1) files to your track's repository. Running either of these scripts will download the latest version of the `configlet` tool to your track's `bin` directory.
+Each track should have a `bin/fetch-configlet` and/or `bin/fetch-configlet.ps1` script. Running these scripts downloads the latest version of configlet to the `bin` directory. You can then use configlet by running `bin/configlet` respectively `bin/configlet.exe`.
```

Comment:
```suggestion
Each track should have a `bin/fetch-configlet` script, and might have a `bin/fetch-configlet.ps1` script too. The first is a bash script, and the second is a PowerShell script.

Running one of these scripts downloads the latest version of configlet to the `bin` directory. You can then use configlet by running `bin/configlet` or `bin/configlet.exe` respectively.
```

This fixes an error:
> by running `bin/configlet` respectively `bin/configlet.exe`

and avoids a possible misreading of "you must run both of these scripts one after the other in order to download configlet".

There aren't any tracks that have just the PowerShell script, right? If so, let's avoid "and/or"?

Please ignore the "The first is a bash script, and the second is a PowerShell script." if you don't like it.